### PR TITLE
Made Agent IDs rewrittable.

### PR DESCRIPTION
**Pull request by xxalpha**
~~Forging is now on CtrlClick~~

It's an alert window.
untitled

### CODE REVIEW

**Comment by Jalleo** [comment]
:secret: :+1:

**Comment by Aranclanos** [comment]
solve metagaming

**Comment by Shadowlight213** [comment]
Just do the same check that the microscanner message does. Have it only he rewritable if they are a traitor. Metagaming fixed.

**Comment by Iamgoofball** [comment]
:+1:

**Comment by WJohn** [comment]
This should've always been the case. :+1:

**Comment by xxalpha** [comment]
Okay, added a check so only traitor minds can use the forging feature.

File: html/changelogs/xxalpha-rw_agent.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+author: xxalpha
+
+delete-after: True
+
+changes: 
+  - tweak: "Agent IDs are now rewrittable. Use Ctrl+Click to forge!"
```
Comment (freerealestate) [inline html/changelogs/xxalpha-rw_agent.yml]: Not that it matters, but it's "rewritable".

**Comment by freerealestate** [comment]
:+1: May finally give this item some use outside of nuke ops

**Comment by Iamgoofball** [comment]
:+1: x :100:

**Comment by Xhuis** [comment]
thumbsup

**Comment by Fox-McCloud** [comment]
Very yes :+1:

**Comment by Aranclanos** [comment]
THINK ON SOMETHING ELSE THAN CTRL CLICK
YOU, READING THIS, PUT A BETTER SUGGESTION. YES YOU

**Comment by Iamgoofball** [comment]
ctrl+click works

**Comment by phil235** [comment]
Since we're already checking for special mind, we could simply use attack_self() and have a window appear to select whether we want to show our ID or rewrite it. Neither action are used frequently so it wouldn't be a big issue.

**Comment by xxalpha** [comment]
Followed phil's suggestion.

**Comment by PKPenguin321** [comment]
iirc unless it was removed, there's a traitor bundle that has a voice changer and like 3 agent IDs, this change makes that bundle kind of silly since you wont need 3
might want to tweak that while you're at it

**Pull request merged**